jenkins/opensearch-dashboards/integ-test-opensearch-dashboards.jenkinsfile

```diff
@@ -0,0 +1,99 @@
+lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
+
+pipeline {
+    agent none
+    parameters {
+        string(
+            defaultValue: '',
+            name: 'opensearch_version',
+            description: 'OpenSearch version to run test on',
+            trim: true
+        )
+
+        string(
+            defaultValue: '',
+            name: 'opensearch_dashboards_version',
+            description: 'OpenSearch Dashboards version to run test on',
+            trim: true
+        )
+        string(
+            defaultValue: '',
+            name: 'opensearch_build_id',
+            description: 'The build number of OpenSearch for which tests should be run',
+            trim: true
+        )
+        string(
+            defaultValue: '',
+            name: 'opensearch_dashboards_build_id',
+            description: 'The build number of OpenSearch Dashboards for which tests should be run',
+            trim: true
+        )
+        string(
+            defaultValue: '',
+            name: 'architecture',
+            description: 'Architecture of the build of OpenSearch/OpenSearch Dashboards',
+            trim: true
+        )
+        string(
+            defaultValue: '',
+            name: 'platform',
+            description: 'Platform of the build of OpenSearch/OpenSearch Dashboards',
+            trim: true
+        )
```

Comment:
I would prefer if the parameters to this job was 1) an OpenSearch Dashboards build URL (e.g. ci.opensearch....) that contains a manifest that would get loaded and all these features such as build ID and architecture and platform were to be extracted, and 2) an OpenSearch URL. Otherwise we keep building assumptions in the URL structure of where things are built, ie. this job "knows" something it shouldn't about the location of builds.

Use CAPS for variables similarly to other Jenkinsfile.